templates/search.html

```diff
@@ -11,7 +11,26 @@
         <!-- Your HTML should start here -->
         <!-- Look up the Jinja documentation to display items passed from the server -->
         <!-- There should be no JavaScript used on this page -->
-
+        <div class="container">
+            <h1>Search</h1>
+            <form class="search" action="/search">
+                <input type="text" name="query" id="query" placeholder="search">
+            </form>
+            <div class="result-meta">Showing {{results|length}} results for "{{searchstring}}"</div>
+            {% for i in results%}
+            <div class="result">
+                {% if 'types' in i %}
+                <a class="title" href="/pokemon/{{i['id']}}">{{i['name']|safe}}</a>
+                <div class="subtitle">Pokemon</div>
+                {% else %}
+                <a class="title" href="/teams/{{i['id']}}">{{i['name']|safe}}</a>
```

Comment:
Similar as my comment above about differentiating types for your different results, could the URL be part of the results object too? That way you could avoid an `if else` block based on type and just render markup based on the properties of the result object.